native_client/ctcdecode/path_trie.h

```diff
@@ -65,7 +63,10 @@ class PathTrie {
   float score;
   float approx_ctc;
   unsigned int character;
-  unsigned int timestep;
+  std::vector<unsigned int> timesteps;
+  // `timesteps_cur` is a temporary storage for each decoding step. 
+  // At the end of a decoding step, it is moved to `timesteps`.
+  std::vector<unsigned int> timesteps_cur; 
```

Comment:
Hm, this feels like it's easy to make mistakes without realizing in the code that handles the PathTrie, but I don't have concrete suggestions for now. Gonna think about this while I do more tests.

Reply:
It is really the same idea as `log_prob_nb_cur` and `log_prob_b_cur`.
The reason I have not named `timesteps` `timesteps_prev` is that it is used outside the decoding loop directly, like `score` on the probabilities side.